test/drafts/Strings.test.js

```diff
@@ -0,0 +1,38 @@
+const Strings = artifacts.require('Strings');
+
+contract('Strings', function ([owner, anotherAccount]) {
+  let strings;
+  const a = 'AZ@#://12345';
+  const b = '$#%#!';
+  const c = '';
+  const ab = a + b;
+  const ac = a + c;
+  const ca = c + a;
+  const zero = '0';
+  const one = '1';
+  const onetrillion = '1000000000000';
+
+  beforeEach(async function () {
+    strings = await Strings.new({ from: owner });
+  });
+
+  describe('String functions', function () {
```

Comment:
This `describe` is unnecessary (the outher `contract` already covers this). I'd go for:

describe concatenate
   it concatenates strings
   it concatenates a string with itself
   it concatenates with the empty string
   it concatenates two empty strings

describe from uint256
  it converts 0
  it converts a positive number
  it converts MAX_UINT256